sidebars.js

```diff
@@ -143,5 +143,10 @@ module.exports = {
     'canaries_automation',
     'FAQs',
     'contact_us',
+    {
+      type: 'category',
+      label: 'Updates for API v4',
```

Comment:
Don't create another section for this, Instead add it into the sections about that relate to the topic